### PR TITLE
Fix vtctldclient command in vtop test script and the fix operator example

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description

This PR fixes a `vtctldclient` command which was incorrectly pushed in the PR https://github.com/vitessio/vitess/pull/12440 while fixing the issue https://github.com/vitessio/vitess/issues/12438

This PR also updates the operator.yaml file and removes `enforceSemiSync` property since it is deleted.
These changes were missed in https://github.com/vitessio/vitess/pull/12440 though they were included in https://github.com/vitessio/vitess/pull/12457

## Related Issue(s)

- https://github.com/vitessio/vitess/issues/12438

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required

### CODE REVIEW

**Comment by GuptaManan100** [comment]
Just as a note, the test in the script `vtop_example.sh` were working before these changes too, but these changes are still good to make since the operator.yaml file includes the crds which are updated.

File: examples/operator/operator.yaml [hunk 1]
```diff
@@ -74,8 +74,35 @@ spec:
                         - kind
                         - name
                       type: object
+                      x-kubernetes-map-type: atomic
```
Comment (rsajwani) [inline examples/operator/operator.yaml:77]: for my knowledge.. Is it some auto generated file ? and what are all these changes?

Comment (GuptaManan100) [reply examples/operator/operator.yaml:77]: It is auto generated, but on vitess-operator. Whenever it changes, we have to copy them here, but I forgot to do that this time. 😭 
Anyways these changes came from upgrading the controller-gen version from `controller-gen.kubebuilder.io/version: v0.6.2` to `controller-gen.kubebuilder.io/version: v0.11.3`

File: test/vtop_example.sh [hunk 2]
```diff
@@ -252,7 +252,7 @@ function verifyVtadminSetup() {
 function verifyVTOrcSetup() {
   # Set the primary tablet to readOnly using the vtctld and wait for VTOrc to repair
   primaryTablet=$(getPrimaryTablet)
-  vtctldclient SetReadOnly "$primaryTablet"
+  vtctldclient SetWritable "$primaryTablet" false
```
Comment (harshit-gangal) [inline test/vtop_example.sh:255]: Do we still have SetReadWrite command as well?

**Review by harshit-gangal — APPROVED** [review]

File: test/vtop_example.sh (hunk 2 above)
Comment (GuptaManan100) [reply test/vtop_example.sh:255]: No we don't. In vtctldclient we only have `SetWritable`.
In vtctlclient we have `SetReadOnly` and `SetReadWrite`

**Review by frouioui — APPROVED** [review]

**Review by rsajwani — COMMENTED** [review]
LGTM

**Pull request merged**